qrenderdoc/Windows/PixelHistoryView.cpp

```diff
@@ -367,6 +367,21 @@ class PixelHistoryItemModel : public QAbstractItemModel
         }
       }
 
+      // Since we change the background color for some cells, also change the foreground color to
+      // ensure contrast with all UI themes
```

Comment:
There's a helper function to do this in a cleaner way, see `contrastingColor` and its use in e.g. `BufferViewer.cpp` for a very similar purpose.